packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -105,14 +112,20 @@ export function publish(options: InitOptions): void {
       flowlet = new ALFlowlet(`ts${captureTimestamp}`, flowlet);
       flowlet = flowletManager.push(flowlet);
 
+      let reactComponentData: ReactComponentData | null = null;
+      if (cacheElementReactInfo) {
+        reactComponentData = getReactComponentData_THIS_CAN_BREAK(element, componentNameValidator);
```

Comment:
This is potentially the only part that can add some perf overhead, but I think for now we can let it happen this way and see what the perf impact is. it is possible that it all would depends on the frequency of these events.